liquibase-standard/src/test/groovy/liquibase/parser/core/yaml/YamlChangeLogParser_RealFile_Test.groovy

```diff
@@ -299,6 +299,15 @@ public class YamlChangeLogParser_RealFile_Test extends Specification {
         assert e.message.startsWith("Syntax error in file liquibase/parser/core/yaml/malformedChangeLog.yaml")
     }
 
+    def "ChangeLogParseException thrown if changelog has two databaseChangeLog tags"() throws Exception {
+        when:
+        new YamlChangeLogParser().parse("liquibase/parser/core/yaml/malformedDoubleChangeLog.yaml", new ChangeLogParameters(), new JUnitResourceAccessor())
+
+        then:
+        def e = thrown(ChangeLogParseException)
+        assert e.message.startsWith("Syntax error in file liquibase/parser/core/yaml/malformedDoubleChangeLog.yaml")
```

Comment:
Does this error message contain more useful information in it?

Reply:
Indeed, it does, I've added an additional assertion on the message. Thanks for the great idea!